### ISSUE TITLE
Cannot resolve configuration property 'preliquibase.continueOnError' 

### ISSUE THREAD

**Issue by landsman** [lbruun/Pre-Liquibase#25]
What I did do wrong?

![Screenshot 2023-11-07 at 23 18 32](https://github.com/lbruun/Pre-Liquibase/assets/3184228/4c97af96-1999-4d6e-9db3-ab221ef711e7)

**Comment by lbruun** [comment]
This is an IDE question. Not per se a question related to PreLiquibase.

Some IDEs have a feature whereby they know "know" (for Spring Boot) which configuration properties are valid. I'm guessing your IDE is IntelliJ IDEA?   The feature in that IDE relies on some metadata to figure this out. For third party libraries you'll often see this message. It doesn't mean you you've spelled anything wrong. It just means that the library author hasn't bothered to publish such metadata along with the library.

I've opened a [feature request](https://github.com/lbruun/Pre-Liquibase/issues/26) on this topic.

Closing this ticket as this is not a bug.
